android/1_download_library.sh

```diff
@@ -124,13 +141,17 @@ download_and_extract http://www.mega-nerd.com/libsndfile/files/libsndfile-1.0.27
 rm -rf speexdsp-1.2rc3 
 download_and_extract http://downloads.xiph.org/releases/speex/speexdsp-1.2rc3.tar.gz
 
-msg "Cloning SDL2"
+# SDL2
 rm -rf SDL/
-hg clone http://hg.libsdl.org/SDL
+download https://hg.libsdl.org/SDL/archive/tip.tar.gz -O SDL.tar.gz --no-timestamping
+mkdir -p SDL
+extract SDL.tar.gz --strip-components=1 -C SDL
 
-msg "Cloning SDL2_mixer"
+# SDL_mixer
 rm -rf SDL_mixer/
-hg clone http://hg.libsdl.org/SDL_mixer
+download https://hg.libsdl.org/SDL_mixer/archive/tip.tar.gz -O SDL_mixer.tar.gz --no-timestamping
+mkdir -p SDL_mixer
+extract SDL_mixer.tar.gz --strip-components=1 -C SDL_mixer
 
```

Comment:
heh, yes could be changed then. Maybe sometime there will get incompatible changes merged in, then we are safe, but I guess we will need to use this again when there is no sdl release the next 2 years :ghost:.